backend/dataall/modules/dataset_sharing/services/dataset_alarm_service.py

```diff
@@ -51,9 +49,9 @@ def trigger_revoke_table_sharing_failure_alarm(
             target_environment: Environment,
     ):
         log.info('Triggering share failure alarm...')
-        subject = f'ALARM: DATAALL Table {table.GlueTableName} Revoking LF permissions Failure Notification'
+        subject = f'Data.all Revoke LF permissions Failure for Table {table.GlueTableName}'[:100]
```

Comment:
update to Permissions to be consistent with capitalization

Reply:
Done!